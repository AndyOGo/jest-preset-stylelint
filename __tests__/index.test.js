const preset = require("../jest-preset.json");
it('should contain expected keys', () => {
  expect(Object.keys(preset)).toMatchSnapshot();
});

it("should match .js extension files", () => {
  expect(preset.moduleFileExtensions).toEqual([
    "js",
    "json",
    "jsx",
    "node"
  ]);
});
