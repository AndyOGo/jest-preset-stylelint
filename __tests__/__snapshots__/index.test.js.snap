// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it contains expected keys 1`] = `
Array [
  "clearMocks",
  "collectCoverage",
  "coverageDirectory",
  "coverageReporters",
  "coverageThreshold",
  "moduleFileExtensions",
  "testEnvironment",
  "testMatch",
  "verbose",
]
`;
